tests/status_buf_spec.lua

```diff
@@ -1,3 +1,5 @@
+-- Ignore line with only whitespace
+-- luacheck: ignore 611
```

Comment:
I really don't like that we have to do this. I think we should disable this by default.

Reply:
I can also just remove the few remaining  lines with whites spaces. The warnings are all in the git commit messages